MonoGame.Framework/Desktop/OpenTKGamePlatform.cs

```diff
@@ -88,13 +88,13 @@ class OpenTKGamePlatform : GamePlatform
 		private OpenALSoundController soundControllerInstance = null;
         // stored the current screen state, so we can check if it has changed.
         private bool isCurrentlyFullScreen = false;
-        private Toolkit toolkit;
         private int isExiting; // int, so we can use Interlocked.Increment
+
+        int windowDelay = 11;
```

Comment:
> Just seems fragile to depend on message pumps that are often inconsistent in timing and order to wait some exact number of updates.

Well on the bright side chances that OpenTK error appearing should also be next to never(I run my game 100 times in Windows, 0 exit errors appeared)

> Also why isn't OpenTK handling this for us? It shouldn't be our job to guess when it is safe to dispose their window.

Actually it is our job, we are disposing of a lot of different OpenGL stuff we created by using the window handle.